Samples/BashTestSpecificToDotNet50/test.json

```diff
@@ -1,12 +0,0 @@
-{
-  "name": "BashTestSpecificToDotNet50",
-  "enabled": true,
-  "requiresSdk": true,
-  "version": "50.0",
```

Comment:
This is a future .NET version. It hasn't been released yet, so it hasn't been EOL'ed either.

Reply:
My mistake, I'll add it back in.